exercises/tournament/tournament_test.rb

```diff
@@ -5,65 +5,64 @@
 
 # Common test data version: cda8f98
 class TournamentTest < Minitest::Test
-
   def test_typical_input
     # skip
     input = <<-INPUT.gsub(/^ */, '')
-      Allegoric Alaskans;Blithering Badgers;win
-      Devastating Donkeys;Courageous Californians;draw
-      Devastating Donkeys;Allegoric Alaskans;win
-      Courageous Californians;Blithering Badgers;loss
-      Blithering Badgers;Devastating Donkeys;loss
-      Allegoric Alaskans;Courageous Californians;win
-    INPUT
+Allegoric Alaskans;Blithering Badgers;win
```

Comment:
> Why should these lines be indented?

It's probably just consistent indent fetish.
Why should any lines be indented?

I guess it's ok if you feel strongly they shouldn't be indented.
In which case remove from the line above the `gsub` that removes the whitespace.

> We wouldn't expect a real problem to change the way its data was configured just so the test appeared correctly indented!

There's no change to the indentation used as an argument to the problem, just its presentation within the tests.